pennylane_lightning/src/simulators/lightning_qubit/utils/LinearAlgebra.hpp

```diff
@@ -758,63 +758,6 @@ inline auto matrixMatProd(const std::vector<std::complex<T>> m_left,
     return m_out;
 }
 
-/**
- * @brief Generate random unitary matrix
- *
- * @tparam PrecisionT Floating point type
- * @tparam RandomEngine Random engine type
- * @param re Random engine instance
- * @param num_qubits Number of qubits
- * @return Generated unitary matrix in row-major format
- */
-template <typename PrecisionT, class RandomEngine>
-auto randomUnitary(RandomEngine &re, size_t num_qubits)
```

Comment:
I assume this wasn't used anywhere?

Reply:
It is, but only for tests and also in LKokkos, so I moved it to `pennylane_lightning/src/utils/TestHelpers.hpp`.